interface/patient_file/encounter/find_code_dynamic.php

```diff
@@ -226,76 +226,83 @@ function SelectList(jobj) {
+                echo "\n";
+                echo "<div class='col'>";
+                echo "<input type='checkbox' name='form_include_inactive' value='1' onclick='oTable.fnDraw()' />" .
+                xlt('Include Inactive') . "\n";
+                echo "\n";
+                echo "<button class='btn btn-secondary btn-sm btn-delete' value='" . xla('Delete') . "' onclick='delcode()'>" . xla('Delete') . "</button>\n";
+                echo "<select name='form_delcodes'>\n";
+                echo " <option value=''>" . xlt('All') . "</option>\n";
+                echo "</select>\n";
+                echo "\n";
+                echo "<button class='btn btn-secondary btn-sm btn-cancel' value='" . xla('Close') . "' onclick='dlgclose()'>" . xla('Close') . "</button>\n";
+                echo "</div>";
             }
-            echo "</select>\n";
-        }
-    } else {
-        echo "<select name='form_code_type' onchange='oTable.fnDraw()'>\n";
-        foreach ($code_types as $key => $value) {
-            echo " <option value='" . attr($key) . "'";
-            echo ">" . xlt($value['label']) . "</option>\n";
-        }
-        echo " <option value='PROD'";
-        echo ">" . xlt("Product") . "</option>\n";
-        echo "   </select>\n";
-    }
-    echo "&nbsp;&nbsp;\n";
-    echo "<input type='checkbox' name='form_include_inactive' value='1' onclick='oTable.fnDraw()' />" .
-    xlt('Include Inactive') . "\n";
-    echo "&nbsp;&nbsp;\n";
-    echo "<input type='button' value='" . xla('Delete') . "' onclick='delcode()' />\n";
-    echo "<select name='form_delcodes'>\n";
-    echo " <option value=''>" . xlt('All') . "</option>\n";
-    echo "</select>\n";
-    echo "&nbsp;&nbsp;\n";
-    echo "<input type='button' value='" . xla('Close') . "' onclick='dlgclose()' />\n";
-}
-if ($what == 'lists') {
-    echo "<input type='button' value='" . xla('Delete') . "' onclick='SelectList({\"code\":\"\"})' />\n";
-}
-echo "</p>\n";
-?>
-
-<table id="my_data_table" class="table table-striped table-hover table-sm" style="width: 90%;">
- <thead>
-  <tr>
-   <th><?php echo xlt('Code'); ?></th>
-   <th><?php echo xlt('Description'); ?></th>
-  </tr>
- </thead>
- <tbody>
-  <tr>
-   <!-- Class "dataTables_empty" is defined in jquery.dataTables.css -->
-   <td colspan="2" class="dataTables_empty">...</td>
-  </tr>
- </tbody>
-</table>
-
-<?php if ($what == 'fields' && $source == 'E') { ?>
-<center>
-<p>
-<input type='text' name='new_field_id' size='20' />&nbsp;
-<input type='button' value='<?php echo xla('Or create this new field ID') ?>' onclick='newField()' />
-</p>
-</center>
-<?php } ?>
-
-</form>
+            if ($what == 'lists') {
+                echo "<button class='btn btn-secondary btn-sm btn-delete' value='" . xla('Delete') . "' onclick='SelectList({\"code\":\"\"})'>" . xla('Delete') . "</button>\n";
+            }
+            echo "</div>\n";
+            ?>
+
+        <table id="my_data_table" class="table table-striped table-hover table-sm">
```

Comment:
Should be surrounded by a "table-responsive" div

Reply:
Done